compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -153,7 +153,11 @@ extern "C" LLVMContextRef LLVMRustContextCreate(bool shouldDiscardNames) {
 
 extern "C" void LLVMRustSetNormalizedTarget(LLVMModuleRef M,
                                             const char *Triple) {
+#if LLVM_VERSION_GE(21, 0)
+  unwrap(M)->setTargetTriple(llvm::Triple(Triple::normalize(Triple)));
```

Comment:
It's annoying that we need to qualify `llvm::` here when the type is already in scope. Let's rename the parameter to separate that, e.g.:
```suggestion
  unwrap(M)->setTargetTriple(Triple(Triple::normalize(Target)));
```